docker/dotnet/Dockerfile

```diff
@@ -18,17 +18,16 @@ FROM debian:11-slim
 LABEL org.opencontainers.image.description="Pulumi CLI container for dotnet"
 WORKDIR /pulumi/projects
 
-ARG DOTNET_VERSION="3.1"
+ARG DOTNET_VERSION="6.0"
```

Comment:
Do you know if this is potentially breaking for users relying on 3.1 in some way?